sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuFileFormatWriter.scala

```diff
@@ -17,7 +17,6 @@
 package org.apache.spark.sql.rapids
 
 import java.util.{Date, UUID}
-
```

Comment:
This whitespace should not have been deleted per our style guidelines.  See https://github.com/NVIDIA/spark-rapids/blob/branch-22.02/docs/dev/idea-code-style-settings.xml#L11-L17

Reply:
Thank you! I have fixed it